locales/fr.json

```
@@ -26,6 +26,7 @@
   "advice:table-presentation": "Tableau de mise en forme",
   "advice:no-href": "Lien sans cible",
   "advice:no-https": "Pas de HTTPS",
+  "advice:select-option-not-empty": "<select> should have empty [value] on first <option>",
```

Translating my suggested english message to french:
```suggestion
  "advice:select-option-not-empty": "Un <select required> devrait débuter par <option value="">",
```

---

Done! Same as before, only change I did was to escape the double quotes.